architecture/README.md

```diff
@@ -33,127 +30,135 @@ types of roles. The TiDB community has the following organizations:
+
+- Get the profile listed on the membership page of the corresponding SIG
 
 ### Reviewer
 
-Reviewers are able to review code for quality and correctness on some part of a
-subproject. They are knowledgeable about both the codebase and software
-engineering principles.
+Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Active Contributors in a specific SIG.
 
-#### How to become a Reviewer?
+Reviewer status is scoped to the part of the codebase under the SIG.
+See `member-list.md` under the specific SIG for a list of Reviewers of the SIG.
 
-* Knowledgeable about the codebase
-* Sponsored by 2 Committers
-* Merged at least 20 PRs 
+#### How to become a Reviewer
+
+- Familiar with the codebase
+- Demonstrated a recoganized level of community engagement in the SIG development, via contributions that have met one or more of the minimal criteria as defined below：
+  - Merged 20+ PRs (may vary among SIGs)
+  - Handled 1+ more issue of Medium difficulty or higher
+- Sponsored by at 2 Committers of the SIG
 
 #### Responsibilities and Privileges
 
-* Responsible for project quality control
-    * Focus on code quality and correctness, including testing and factoring 
-    * May also review for more holistic issues
-* Expected to be responsive to review PR on time
-* Assigned test and fix bugs in subproject 
-* Have the rights to approve PR
-* Participate in the future seminar
+- Ensure the correctness of the code submitted to the SIG
+- Respond promptly to PRs
+- Participate in SIG meetings regularly
+- Approve code reviews with LGTM (Looks Good To Me)
+- Get the profile listed on the membership page of the corresponding SIG
 
 ### Committer
 
-Committers are developers of the projects and are elected from contributors.
-Currently.
+Committers come from Reviewers, and are recoganized by Maintainers and SIG Tech Leads for their outstanding contributions to the specific SIG.
 
-#### How to become a Committer?
+Committer is scoped to the specific SIG where the promotion applies. See `member-list.md` under the specific SIG for a list of Committers of the SIG.
 
-A new Committer must be nominated by any existing Maintainer or PMC member, and
-is generated through the voting of PMC members.
+#### How to become a Committer
 
 A Committer must have accomplished one or more of the following items:
 
-* Your commits have been approved and merged by Maintainers for over ten times.
-* You have implemented a complete feature.
-* You have fixed a vital bug.
+- Demonstrated a good sense of responsibility in PR reviews within the SIG. The review contributions must meet the amount or the difficulty criteria as defined in the charter of the specific SIG.
+- Demonstrated deep understanding of the TiDB components specific to the SIG by contributing significantly as:
+    - Finished 2+ tasks of Medium difficulty or higher
+    - Fixed 1+ task of Hard difficulty
```

Comment:
`1+` means `one or larger than one` or `larger than one`?

Reply:
Good point. I updated it to "1 or more"